lib/html.js

```diff
@@ -29,7 +29,7 @@ module.exports = (methods, routes) => {
       </tr>
       ${routes.map(r => `
         <tr>
-          <td>${r.path}</td>
+          <td><a href="${r.path}">${r.path}</a></td>
```

Comment:
Is that href right?

Reply:
This looks/works right in my browser, but do we want the server address here as well instead of just relative path?

Reply:
how does it work?  you are going to a page that doesn't exist

Reply:
....